frontend/src/types/Mesh.ts

```diff
@@ -92,6 +92,29 @@ export interface MeshDefinition {
   timestamp: number;
 }
 
+export interface Tag {
+  name: string;
+}
+
+export interface ControlPlaneConfig {
+  OutboundTrafficPolicy?: any;
+  configMap?: any;
```

Comment:
```suggestion
  configMap?: any;
  outboundTrafficPolicy?: any;
```
alphabetization and capitalization